app/views/campaign_pages/_link_form.slim

```diff
@@ -0,0 +1,28 @@
+.collection-editor
+  ul.list-group
+    - page.links.each do |link|
+      = render 'link', link: link
+
+  = form_for Link.new, remote: true, html: {class: 'form-element', id: 'new_collection_element'}  do |f|
+
+    = f.hidden_field :campaign_page_id, value: page.id
+
+    .col-md-12
+      .form-group.col-md-3
+        = f.label :url, "URL"
```

Comment:
Yeah. Something I'm realizing about our i18n is that it's mostly the Champaign back end, which is used internally. It won't be too hard, but we're going to have to do a little thinking to pass all our localizations through the liquid rendering.